autogpt_platform/autogpt_libs/autogpt_libs/supabase_integration_credentials_store/store.py

```diff
@@ -14,26 +17,29 @@
 
 
 class SupabaseIntegrationCredentialsStore:
+    locks = KeyedMutex()
```

Comment:
FYI: This will be gone very soon: https://github.com/Significant-Gravitas/AutoGPT/pull/8197/files#diff-39d77f5b792b124e499a265155ab201f0a211a16c57129cec51d32ba4f0c695d

Let me know if there will be any issue migrating this to Redis.

Reply:
It shouldn't be an issue, other than extra time spent. I just need the locking service to be available within `autogpt_libs`, while you're keeping it within `backend`.